src/contexts/Api/index.tsx

```diff
@@ -37,12 +38,16 @@ export const APIProvider = ({ children }: { children: React.ReactNode }) => {
   const [api, setApi] = useState<ApiPromise | null>(null);
 
   // network state
-  const _name: NetworkName =
-    (localStorage.getItem('network') as NetworkName) ?? NetworkName.Polkadot;
+  const _name = localStorage.getItem('network') as NetworkName;
+  const initialMeta = NETWORKS[_name] || NETWORKS[DEFAULT_NETWORK];
+
+  if (!_name || !NETWORKS[_name]) {
+    localStorage.setItem('network', DEFAULT_NETWORK);
```

Comment:
Do you want to mention that it sets the default network value to the local storage in the changelog?

Reply:
Thank you! I phrased it as follows:
_Improved network initialization and error handling to prevent application crashes resulting from incorrect localStorage network. Added a default network for this scenario._